File: Python_code/testForString.py
import suds
from suds.client import Client
username='u1xxxxx'
pwd = 'xxxxxx'
url = 'http://service.ygys.net/resumeservice.asmx?WSDL'
client = Client(url)
result = client.service.TransResume(username,pwd,'name:wangbing mobile:13890988878')

print result

File: Nodejs_code/testForString.js
var soap = require('soap');
var url = 'http://service.ygys.net/resumeservice.asmx?WSDL';
var args = { username: 'u100xxx', pwd: 'xxx', original: '姓名：张三 性别：男 手机：13890009900 学历：本科 毕业院校：清华大学 工作经验：3年' };

soap.createClient(url,function(err,client) {
  if (err) {
    console.log(err);
  }

  client.ResumeService.ResumeServiceSoap12.TransResume(args, function(err, result){
    console.log(result);
  }); 

});
